### PR TITLE
Emoticons Cheat sheet: Added more emoticons for happy face

### PR DESCRIPTION
**Pull request by mayank7**
Added more emoticons for happy face to emoticons cheat sheet

IA Page: https://duck.co/ia/view/emoticon_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Emoticon Cheat Sheet](https://duck.co/ia/view/emoticon_cheat_sheet)

**Description**: Displays a quick reference for some basic emoticons.

**Example Query**: [Emoticon Cheat Sheet](https://beta.duckduckgo.com/?q=Emoticon Cheat Sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/emoticon_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/emoticon_cheat_sheet)*

File: share/goodie/cheat_sheets/json/emoticon.json [hunk 1]
```diff
@@ -24,6 +24,12 @@
         }, {
             "val": "Enamored",
             "key": ":3"
+        }, {
+            "val": "Face with goggles",
+            "key": "8-)"
+        }, {
+            "val": "Face with black goggles",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/emoticon.json]: `black goggles`? I'm not sure this is a very *happy* face.

**Comment by mayank7** [comment]
@GuiltyDolphin I removed Face with black goggle from Happy section

**Comment by mayank7** [comment]
@GuiltyDolphin I removed Face with black goggles from Happy section and it failed in  continuous-integration/travis-ci/pr . 
I am not getting why that happened, can you please review and help me.

**Comment by GuiltyDolphin** [comment]
@mayank7 Thanks, looking good! :dolphin:

**Pull request merged**